internal/mtail/mtail.go

```diff
@@ -344,7 +345,7 @@ func (m *Server) Run() error {
 			return err
 		}
 	} else {
-		m.store.StartExpiryLoop()
+		m.store.StartExpiryLoop(*m.storeExpireTickInterval)
```

Comment:
The test failure from CI is because this is nil when run in test.
```suggestion
                if (*m.storeExpireTickInterval != nil ) {
		  m.store.StartExpiryLoop(*m.storeExpireTickInterval)
		}
```